socorro/submitter/breakpad_submitter_utilities.py

```diff
@@ -1,68 +1,58 @@
-#! /usr/bin/env python
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/
-from configman import Namespace
-from socorro.external.crashstorage_base import CrashStorageBase
 
 import os
 import urllib2
+
+from configman import Namespace
 import poster
+
+from socorro.external.crashstorage_base import CrashStorageBase
+
+
 poster.streaminghttp.register_openers()
 
 
 class BreakpadPOSTDestination(CrashStorageBase):
-    """This a crashstorage derivative that pushes a crash out to a
-    Socorro collector waiting at a url.
+    """This a crashstorage derivative that pushes a crash out to a Socorro collector waiting at a url.
 
     """
     required_config = Namespace()
     required_config.add_option(
-        'url',
-        short_form='u',
-        doc="The url of the Socorro collector to submit to",
-        default="http://127.0.0.1:8882/submit"
+        'urls',
+        doc='One or more urls to submit to separated by commas',
+        default="http://127.0.0.1:8888/submit"
     )
-    required_config.add_option(
-        'echo_response',
-        short_form='e',
-        doc="echo the submission response to stdout",
-        default=False
-    )
-
-    def __init__(self, config, quit_check_callback=None):
-        super(BreakpadPOSTDestination, self).__init__(
-            config,
-            quit_check_callback
-        )
-        self.hang_id_cache = dict()
```

Comment:
The sole purpose of overriding `__init__` is to initialize `self.hang_id_cache`, but that never gets used, so I nixed this whole thing.